tenx-tech/stargazer
# save screenshots as files ;)

### What does this PR do?
Save stargazer screenshots as files

### Where should the reviewer start?
Run both stargazer server and on mobile and also web view

### How should this be tested?
This PR doesn't change stargazer/app (mobile client) only the web viewer and server so you can run start stargazer from this repository and run stargazer from `tenx-react-native` you will notice the `{os}-data.json` file has image path instead of the base64 encoding and a new `screenshots` directory created next to it has the actual images 
 - please make sure to do `npm install` before testing

### Screenshots / Screen capture if any
![Screenshot 2019-08-07 at 9 37 20 PM](https://user-images.githubusercontent.com/5216291/62632223-b66d5f00-b964-11e9-9e5e-a9ab8c7b9daa.png)
![Screenshot 2019-08-07 at 9 37 28 PM](https://user-images.githubusercontent.com/5216291/62632224-b66d5f00-b964-11e9-9577-cb2c4a5aa28c.png)
